doc/articles/features/windows-devices-haptics.md

```diff
@@ -11,10 +11,22 @@ uid: Uno.Features.WDHaptics
 
 ## `VibrationDevice` class
 
-The `RequestAccessAsync` method is implemented on all platforms and returns `Allowed` on all platforms, except for Android and Tizen. In the case of Android, the `android.permission.VIBRATE` permission needs to be declared. In the case of Tizen, the `http://tizen.org/privilege/haptic` privilege needs to be declared.
+### Platform-specific requirements
 
 The `GetDefaultAsync` method is implemented on all platforms and returns `null` for the unsupported platforms (WPF, GTK).
 
+#### Android
+
+For Android, there are is one permission you must configure before using this API in your project. To do that, add the following to `AndroidManifest.xml`:
+
+```xml
+<uses-permission android:name="android.permission.VIBRATE" />
+```
+
+#### Tizen
+
+In the case of Tizen, the `http://tizen.org/privilege/haptic` privilege needs to be declared in `config.xml` file.
```

Comment:
Is it `config.xml` or `tizen-manifest.xml`?

Reply:
I have never worked with Tizen before, so when I was looking at the docs, I found only config.xml, but when I search for the tizen-manifest.xml specifically, I get a lot of results, it's probably that one.